code/modules/admin/secrets/fun_secrets/triple_ai_mode.dm

```diff
@@ -2,7 +2,7 @@
 	name = "Triple AI Mode"
 
 /datum/admin_secret_item/fun_secret/triple_ai_mode/can_execute(var/mob/user)
-	if(ticker && ticker.current_state > GAME_STATE_PREGAME)
+	if(SSticker && SSticker.current_state > GAME_STATE_PREGAME)
```

Comment:
Could inline return this as 
return (SSticker && SSticker.current_state > GAME_STATE_PREGAME) && ..()

Reply:
> Could inline return this as return (SSticker && SSticker.current_state > GAME_STATE_PREGAME) && ..()

Ready